robot_smach_states/test/manipulation/test_manipulation.py

```diff
@@ -11,6 +11,17 @@
 from robot_smach_states.util import designators as ds
 
 
+def assert_any_first_argument(unit_test_case, calls, argument):
+    for call in calls:
+        if call[0][0] == argument:
```

Comment:
Are you sure, that `call[0][0]` is always available? Aka do the list always have at least one element in them?

Reply:
No, if the call has no arguments then call[0] is an empty list. But since this is a test it is allowed to fail. the test will still give an accurate result

Reply:
Might be helpful to include this in option in the function to raise a more accurate error?

Reply:
Will do